estate/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Real Estate",
+    "version": "1.0",
+    "description": "Real Estate Management System",
+    "summary": "Real Estate Management System",
```

Comment:
```suggestion
    'name': 'Real Estate',
    'version': '1.0',
    'description': "Real Estate Management System",
    'summary': 'Real Estate Management System',
```

Prefer using double quotes ("") for long strings, multiline or user inputs/displays.